modules/ztp-removing-content-from-managed-clusters.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-removing-content-from-managed-clusters_{context}"]
+= Removing content from a CR in managed clusters
+
+You can remove content from a custom resource (CR) deployed to a managed cluster through a policy.
+
+By default, all `Policy` CRs created from a `PolicyGenTemplate` CR have a `complianceType` set to `"musthave"`.
+In this mode, when you remove a field from a CR, the value is removed from the policy but is not removed from the CR on the managed cluster.
+A `"musthave"` policy without the removed field is still compliant because the CR on the managed cluster has all the other content.
```

Comment:
I might suggest being slightly more specific:
```suggestion
A `"musthave"` policy without the removed field is still compliant because the CR on the managed cluster has all the specified content.
```